infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/JsonUtil.java

```diff
@@ -153,6 +165,25 @@ private static <T> Optional<T> getAttributeFromParser(
     return Optional.empty();
   }
 
+  private static <T> Optional<T> getFirstAttributeFromParser(
```

Comment:
this is more like finding the first attribute matching a specified name at any depth... I'm not sure the name chosen really indicates that...
`findAttributeFromParser`

I'm not sure what a good name is, but i don't think 'getFirstAttributeFromParser' really captures it...

Reply:
Would be interested why we can't specify a path? it's kind of scary we're just doing a full scan where we should know a path...

Reply:
I agree, a whole scan could be dangerous. I rolled it back to the initial version I came up with: hard code the paths (message -> slot, signed_block -> message -> slot and blinded_block -> message -> slot).